conversion_helpers_test.go

```diff
@@ -951,6 +952,24 @@ var _ = Describe("Resources", func() {
 						}))
 					})
 				})
+
+				Context("when volumeMountedFiles are present", func() {
+					BeforeEach(func() {
+						task.VolumeMountedFiles = []*models.File{{
+							Path:    "/tmp/some/path",
+							Content: "file content",
+						}}
+					})
+
+					It("allowing task to uses volumeMountedFiles", func() {
```

Comment:
❓ Is there a way to actually use volumeMountedFiles for a task? How does one enable that via the CC API?

Reply:
Thank @beyhan ! I didn't know you could run a task on an app 🤔 !